src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlAssertion.cs

```diff
@@ -64,129 +79,124 @@ IEnumerable<SamlStatement> samlStatements
             //    throw LogHelper.ExceptionUtility.ThrowHelperArgument(SR.GetString(SR.SAMLAssertionIDIsInvalid, assertionId));
```

Comment:
remove commented code, open issue asking if runtime should validate assertion id